TESTING.md

```diff
@@ -0,0 +1,293 @@
+    * [GOSS Templates](#goss-templates)
+    * [Composition](#composition)
+  * [Run GOSS locally](#run-goss-locally)
+  * [Specific GOSS acceptance criteria](#specific-goss-acceptance-criteria)
+
+## Where to find the tests
+
+All the apps have an associated folder inside [/.vib](https://github.com/bitnami/containers/tree/main/.vib) with their custom tests implementation (the `goss` subfolder) and files containing their test plans (`vib-verify.json` and `vib-publish.json`).
+
+## VMware Image Builder (VIB)
+
+The service that powers the verification of the thousands of monthly tests performed in the repository is VMware Image Builder. [VMware Image Builder](https://tanzu.vmware.com/content/blog/how-bitnami-uses-vmware-image-builder-to-deploy-apps) (VIB) is a platform-agnostic, API-first modular service that allows large enterprises and independent software vendors to automate the packaging, **verification**, and publishing processes of software artifacts on any platform and cloud.
+
+## VIB pipeline files
+
+The CI/CD pipelines in the repository are configured to trigger VIB when an app needs to be verified. But as every application is different, VIB needs to be supplied with a definition of the set of actions and configurations that precisely describe the verification process to perform in each case. This is the role of the aforementioned `vib-verify.json` and `vib-publish.json` files, which every app defines and can be found alongside its tests inside the `/.vib` folder.
+
+Let's take a look at an example and try to understand it!
+
+```json
+{
+  "context": {
+    "resources": {
+      "url": "{SHA_ARCHIVE}",
+      "path": "{VIB_ENV_PATH}"
+    },
+    "runtime_parameters": "Y29tbWFuZDogWyJ0YWlsIiwgIi1mIiwgIi9kZXYvbnVsbCJd"
+  },
+  ...
+  "phases": {
+    ...
+    "verify": {
+      "actions": [
+        {
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib"
+            },
+            "tests_file": "wordpress/goss/goss.yaml",
+            "vars_file": "wordpress/goss/vars.yaml",
+            "remote": {
+              "pod": {
+                "workload": "deploy-wordpress"
+              }
+            }
+          }
+        },
+        {
+          "action_id": "trivy",
+          "params": {
+            "threshold": "CRITICAL",
+            "vuln_type": [
+              "OS"
+            ]
+          }
+        },
+        ...
+      ]
+    }
+  }
+}
+```
+
+This guide will focus on the `verify` phase section, of which there are some things to remark on:
+
+* For the testing of containers, VIB will take the built container in the previous `package` phase and include it in a basic Helm chart template composed of a deployment and service template. Beware VIB will only be taking into account the docker image `ENTRYPOINT/CMD` command of the image. Consequently, this both simplifies and limits the configurability of the template chart and container image tested underneath.
```

Comment:
I'd separate this into two points: one that explains that a chart template is used and another one for the entrypoint.

```suggestion
* For the testing of containers, VIB will take the built container in the previous `package` phase and include it in a basic Helm chart template composed of a deployment and service template. 

* VIB does only allow to modify the `ENTRYPOINT/CMD` of the image (through `runtime_parameters`). Consequently, this both simplifies and limits the configurability of the template chart and container image tested underneath.
```